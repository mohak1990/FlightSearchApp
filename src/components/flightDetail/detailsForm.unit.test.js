import React from 'react';
import DetailsForm from './detailsForm';
import { mount, shallow } from 'enzyme';
import ShallowRenderer from 'react-test-renderer/shallow'

test('Snapshot testing for DetailsForm component', () => {
  const renderer = new ShallowRenderer()
  const result = renderer.render(<DetailsForm origin = "" dest = "" />)
  expect(result).toMatchSnapshot()
});
