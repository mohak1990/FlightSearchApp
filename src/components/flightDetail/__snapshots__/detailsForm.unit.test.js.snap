// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing for DetailsForm component 1`] = `
<div>
  <div
    className="flight_detail"
  >
    <span
      className="flight_detail--items flight_detail--items--img"
    >
      <img
        alt="flightimg"
        className="flight_detail--image"
        src="test-file-stub"
      />
    </span>
    <span
      className="flight_detail--items"
    >
      <div
        className="flight_detail--items--info"
      />
      <div
        className="flight_detail--items--info--about"
      />
    </span>
    <span
      className="flight_detail--items"
    >
      <div
        className="flight_detail--items--info"
      />
      <div
        className="flight_detail--items--info--about"
      >
        
      </div>
    </span>
    <span
      className="flight_detail--items"
    >
      <div
        className="flight_detail--items--info"
      />
      <div
        className="flight_detail--items--info--about"
      >
        
      </div>
    </span>
    <span
      className="flight_detail--items flight_detail--items--timeTaken"
    >
      <div
        className="flight_detail--items--info flight_detail--items--info"
      />
      <div
        className="flight_detail--items--info--about"
      >
        Non-Stop
      </div>
    </span>
    <span
      className="flight_detail--items flight_detail--items--amount"
    >
      <div
        className="flight_detail--items--info "
      >
        â‚¹ 
      </div>
    </span>
  </div>
</div>
`;
